docker/docker-compose.override.yml

```diff
@@ -31,8 +31,13 @@ services:
       - mysql
     environment:
       - DATAHUB_TELEMETRY_ENABLED=${DATAHUB_TELEMETRY_ENABLED:-true}
+      - AUTH_POLICIES_ENABLED=${AUTH_POLICIES_ENABLED:-true}
```

Comment:
Please remove from here

Reply:
I removed the default value, however, we need these environment variables here so that they get added in the quickstart docker-compose file, otherwise, the customer wouldn't be able to set these variables from the command line while going through the quick start